pkgs/by-name/dw/dwl/package.nix

```diff
@@ -0,0 +1,125 @@
+{
+  lib,
+  fetchFromGitea,
+  installShellFiles,
+  libX11,
+  libinput,
+  libxcb,
+  libxkbcommon,
+  pixman,
+  pkg-config,
+  stdenv,
+  testers,
+  wayland,
+  wayland-protocols,
+  wayland-scanner,
+  wlroots,
```

Comment:
dwl 0.6 depends on wlroots 0.17, the default one will be bumped to 0.18 by #330630, so dwl won't build after both branches are merged.

Reply:
The default is set at all-packages.nix.